docs/tutorials/FlowTutorial.md

```diff
@@ -50,16 +50,6 @@ Use the `bash` shell to run commands and scripts.
 Install OpenROAD Flow Scripts. Refer to the [Getting Started with
 OpenROAD Flow](https://openroad-flow-scripts.readthedocs.io/en/latest/user/GettingStarted.html) documentation.
 
-If ORFS is already installed but needs updating, run the
-following commands:
-
-```
-cd OpenROAD-flow-scripts
-git checkout master
-git pull
-git submodule update
-```
-
```

Comment:
Keep the lines 53-62 untouched. I will modify in my PR https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1145  by linking your FAQs.md for updating ORFS if already installed. @luarss

Reply:
Understood. Will revert these for now